packages/trader/src/Modules/Trading/Containers/__tests__/radio-group-options-modal.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../trader-providers';
+import RadioGroupOptionsModal from '../radio-group-options-modal';
+
+const default_mocked_props = {
+    is_open: true,
+    modal_title: 'tests title',
+    toggleModal: jest.fn(),
+};
+const default_mock_store = {
+    modules: {
+        trade: {
+            accumulator_range_list: [1, 2, 3],
+            growth_rate: 0.03,
+            onChange: jest.fn(),
+            tick_size_barrier: 0,
+            proposal_info: {},
+        },
+    },
+};
+
+jest.mock('Modules/Trading/Containers/Multiplier/multiplier-options', () =>
+    jest.fn(() => <div>MultiplierOptions component</div>)
+);
+jest.mock('Modules/Trading/Components/Form/RadioGroupWithInfoMobile', () =>
+    jest.fn(props => <div data-test={props.has_error_or_not_loaded}>RadioGroupWithInfoMobile component</div>)
```

Comment:
@kate-deriv our naming convention tells us that for props we use `snake_case` like this: `data_test`. please check all places with this prop